server/migrations/20200827191721-reaction-subscriptions.js

```diff
@@ -0,0 +1,35 @@
+/* eslint-disable quotes */
```

Comment:
Can we have a name for this migration that says what is happening, instead of just what is being affected? (verb + noun instead of just noun, in general, would be best practice in many parts of our codebase where we are defining actions)